cmd/neofs-lens/internal/object/get.go

```diff
@@ -0,0 +1,38 @@
+package object
+
+import (
+	common "github.com/nspcc-dev/neofs-node/cmd/neofs-lens/internal"
+	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/engine"
+	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
+	"github.com/spf13/cobra"
+)
+
+var getObjectCMD = &cobra.Command{
+	Use:   "get",
+	Short: "Get object from NeoFS",
+	Long:  "Get object from NeoFS",
+	Args:  cobra.NoArgs,
+	Run:   getObject}
+
+func init() {
+	common.AddAddressFlag(getObjectCMD, &vAddress)
+	common.AddOutputFileFlag(getObjectCMD, &vOut)
+	common.AddConfigFileFlag(getObjectCMD, &vConfig)
+}
```

Comment:
```suggestion
}

```

btw, what IDE do you use? i think most of that should be formatted automatically (even by the `go fmt`, not by the IDE itself)

Reply:
goland

Reply:
ok, most style problems should be able to be fixed automatically via `go fmt` file watcher then